gprofiler/utils/__init__.py

```diff
@@ -265,11 +279,23 @@ def run_process(
         raise CalledProcessError(retcode, process.args, output=stdout, stderr=stderr)
     return result
 
+def pgrep_keyword_exe(match: str) -> List[Process]:
```

Comment:
Put this definition under `if is_windows()`

Reply:
Why `keyword` in the name?

Reply:
Done. Removed the keyword as well.